stock_request/models/stock_request.py

```diff
@@ -19,7 +19,7 @@ def _get_default_requested_by(self):
     def _get_expected_date():
         return fields.Datetime.now()
 
-    name = fields.Char(states={"draft": [("readonly", False)]})
+    name = fields.Char(readonly=True)
```

Comment:
Please remove `readonly=True` here, as it won't be importable if not, and put it in the view.

Reply:
@pedrobaeza , I removed the `readonly` attribute from the Python code, as it already exists in the view as `readonly="1"`. I believe this was causing confusion and wasn't working in the previous version, as this field should always be readonly.
Please let me know if the change is OK.

img from V16
![image](https://github.com/OCA/stock-logistics-request/assets/7775116/5b0535fb-5dfd-4ca2-8b61-ad839b0b28d4)